backend/src/game/game.ts

```diff
@@ -0,0 +1,48 @@
+import { Server, Socket } from 'socket.io';
+import { CreateGameDto, StateDto } from '@dtos/game';
+import { GameState } from './game-state';
+const FRAMERATE = 30;
+
+export class Game {
+  gameId: number;
+  players: [Socket | undefined, Socket | undefined];
+  server: Server;
+  room: string;
+  state: GameState;
+  updateInterval: NodeJS.Timer;
+
+  constructor(init: CreateGameDto, server: Server) {
```

Comment:
```suggestion
  constructor({ gameId: id }, server: Server) {
```
If I am correct `CreateGameDto` is unused elsewhere and could be safely deleted (it is not a Data Transfer Object between the front and the back).

Reply:
I created #251 . It is a simple change that does not require this PR to be postponed.